functions/src/ro/sci/domain/Cauciuc.java

```diff
@@ -0,0 +1,20 @@
+package ro.sci.domain;
+
+
+public class Cauciuc {
+    public int wide;
+    public int hight;
+    public int round;
+    public boolean winter;
+    public String maker;
+
+    public Cauciuc(int wide, int hight, int round, boolean winter, String maker) {
```

Comment:
daca mai ai vreo douaj' de field-uri, cum o sa faci ?
am pus pe grup un articol despre getters and setters, da-i o sansa :D

Reply:
cu Builder Inner Class :)

am vazut la Andrei :)